topi/python/topi/cuda/conv2d.py

```diff
@@ -67,7 +67,7 @@ def _callback(op):
 
 @autotvm.register_topi_compute("conv2d_cudnn.cuda")
 def conv2d_cudnn(cfg, data, kernel, strides, padding, dilation, layout='NCHW',
-                 out_dtype='float32'):
+                 out_dtype='float32', groups=1):
```

Comment:
move `groups` before `out_dtype`

Reply:
Any good reason. This is the first parameter with a default value.

Reply:
That's just my suggestion. I feel `groups` is more common than `out_dtype`. Another reason is that `group_conv2d` also puts `groups` before `out_dtype`. It'll be more consistent.